src/Formplot/FileFormat/FormplotHelper.cs

```diff
@@ -27,8 +27,9 @@ public static Version GetFileFormatVersion( FormplotTypes formplotType )
 		{
 			return formplotType switch
 			{
-				FormplotTypes.None => new Version( 1, 0 ),
-				_                  => new Version( 2, 0 )
+				FormplotTypes.None   => new Version( 1, 0 ),
```

Comment:
Kleinigkeit: Bin es gewohnt, dass Fälle wie None/Empty als letzte vor Default gelistet werden.

Reply:
"None" ist ein unpassender Name. Dabei handelt es sich um die ".prpx" Dateien, die nur Properties haben. Leider können wir den Namen nicht mehr ändern, da er so in unzähligen Dateien serialisiert ist.